packages/server/src/channels/base/conduit.ts

```diff
@@ -96,9 +97,22 @@ export abstract class ConduitInstance<TConfig, TContext extends ChannelContext<a
     }
 
     const { userId, conversationId } = await this.app.mapping.getMapping(clientId, conduit.channelId, endpoint)
-    return this.app.messages.create(conversationId, userId, endpoint.content, {
-      conduit: { id: this.conduitId, endpoint: _.omit(endpoint, 'content') }
-    })
+
+    if (started) {
+      await this.app.stream.stream(
+        'conversation.started',
+        { conversationId, channel: channel.name },
+        clientId,
+        userId,
+        {
+          conduit: { id: this.conduitId, endpoint }
+        }
+      )
+    } else {
+      return this.app.messages.create(conversationId, userId, endpoint.content, {
+        conduit: { id: this.conduitId, endpoint: _.omit(endpoint, 'content') }
+      })
+    }
```

Comment:
Not the cleanest implementation. Channels are going to be refactored when they are moved to their own package and this is going to be way better